src/docs/product/issues/issue-details/performance-issues/consecutive-http/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "Consecutive HTTP"
+sidebar_order: 30
+redirect_from:
+  - /product/issues/performance-issues/consecutive-http/
+description: "Learn more about Consecutive HTTP and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+Consecutive HTTP issues are created when a set of sequential HTTP spans consistently exceeds 1000ms.
+
+## Detection Criteria
+
+<Note>
+
+This issue is currently only available for backend and mobile SDKs.
+
+</Note>
+
+The detector for this performance issue looks for a set of sequential, non-overlapping HTTP spans. It intentionally ignores non-HTTP spans when finding this set because the browser commonly performs other asynchronous operations, such as loading resources between HTTP spans.
+
```

Comment:
```suggestion
The detector for this performance issue looks for a set of sequential, non-overlapping HTTP spans. It intentionally ignores non-HTTP spans because the browser commonly performs other asynchronous operations, such as loading resources between HTTP spans.

```